roles/mockturtle/files/sopel_modules/invite.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+# vi: set fileencoding=utf-8
+
+import time
+import sopel.bot as bot
+import sopel
+
+@sopel.module.rule(r'.*[Ii]nvite.*')
+def invite(bot, trigger):
+    """To new members that are looking for an invitation"""
+    joinedTime = bot.db.get_nick_value(trigger.nick, 'join_channel')
+    diff = time.gmtime() - joinedTime
+    if diff < 3600:
+        bot.reply(
+            "New contributors are always welcome on Lobsters. If you are the
+author or otherwise involved with a story that was submitted to the
+site, ask and someone will invite you. If not, offer some good links
+or thoughts on a story and someone will likely invite you so can post
+them.  Alternatively, share a link to your homepage or public profile
+(GitHub, Twitter, YC News and LinkedIn are popular choices) so people
+can see you're not a spammer. https://lobste.rs/chat")
+
+@sopel.module.event('JOIN')
+@priority('low')
+@unblockable
+def note(bot, trigger):
+    if not trigger.is_privmsg:
+        bot.db.set_nick_value(trigger.nick, 'join_channel', time.gmtime())
```

Comment:
Let's use trigger.sender here (the channel name) as part of the value.  The seen module works a little differently in this regard as it reports cross-channel activity.  Here we want to know they joined the same channel than they triggered in.  This will required modification to where the joinedTime is looked up too.

Reply:
I was not clear enough here, I'm sorry.  I meant to say, instead of setting the nick value of "join_channel" to set the nick value of `"join_channel_" + trigger.sender`.

Making the channel name part of the value.  Then, on the other end, when you look it up:

`joinedTime = bot.db.get_nick_value(trigger.nick, 'join_channel_' + trigger.sender)`

In effect keeping a separate joined time for each channel the bot is in, and looking up the correct one when the trigger is called.

Reply:
That's a good way of discerning joined channels. Fixing now